src/content/migrate/4.md

```diff
@@ -156,3 +165,10 @@ module.exports = {
 ## module.loaders
 
 `module.loaders` were deprecated since webpack 2 and are now removed in favor of [`module.rules`](/configuration/module/#rule).
+
+
+## Additional Reading
```

Comment:
please move both of these links to the head of the file under line 9 in `related:` section.